addons/curator-link/addon.json

```diff
@@ -23,16 +34,35 @@
     }
   ],
   "settings": [
+    {
+      "id": "mentions",
+      "name": "Make forum mentions clickable",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "id": "curator",
+      "name": "Make the cuator's username clickable",
+      "type": "boolean",
+      "default": true
+    },
     {
       "id": "styleAsNormalText",
       "name": "Style as normal text",
```

Comment:
Users might think that this means to style both the FPC's username and mentions as normal text. While this setting goes away when you have the `curator` setting disabled, that setting is enabled by default, so unless users change it, they won't know the `styleAsNormalText` setting has to do with the curator link, so we might want to have that `curator` setting disabled by default.

Reply:
Also, "cu***r***ator" is typoed.